pkg/helper/operators.go

```diff
@@ -0,0 +1,40 @@
+package helper
+
+import (
+	"log"
+
+	osconfig "github.com/openshift/client-go/config/clientset/versioned"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// CheckOperatorHealth attempts to look at the state of all operator and returns true if things are healthy.
+func CheckOperatorHealth(oscfg *osconfig.Clientset) (bool, error) {
+	fail := true
```

Comment:
Let's name this "success," since this more accurately reflects how this is being used. e.g. "success is true unless it's set to false by the later check.

Reply:
I'll nit further and change it to `CheckOperatorReadiness`